datalad/cmdline/main.py

```diff
@@ -180,6 +189,9 @@ def main(cmdlineargs=None):
         pass
     # parse cmd args
     cmdlineargs = parser.parse_args(cmdlineargs)
+    if not cmdlineargs.change_path is None:
+        for path in cmdlineargs.change_path:
+            os.chdir(path)
```

Comment:
No, if it blows up anywhere, we want to be in that directory. If datalad itself terminates the parent shell will still be where it was. I guess that could be a test...